meta-ros-common/conf/ros-distro/include/enable-fortran.inc

```diff
@@ -1,9 +1,9 @@
-# Copyright (c) 2019 LG Electronics, Inc.
+# Copyright (c) 2019-2020 LG Electronics, Inc.
 
 # FORTRAN support (copied from conf/local.conf.sample.extended)
 # Enabling FORTRAN
 # Note this is not officially supported and is just illustrated here to
```

Comment:
`Note this is not officially supported and is just illustrated here to show an example of how it can be done` -> `Note this is not officially supported by OpenEmbedded`.

Reply:
Is it worth to deviate from the conf/local.conf.sample.extended? If you think it should be re-phrased, then patching it in oe-core first would be IMHO better.

Reply:
It's not clear that which lines have been copied verbatim from `conf/local.conf.sample.extended`, so the reader will probably think that we've added the comments.